1_sudoku/board.py

```diff
@@ -124,6 +131,10 @@ def num_solved_boxes(board_dict: Dict[str, str]) -> int:
     def sorted_box_possibilities(board_dict: Dict[str, str]) -> List[str]:
         options = [box for box in board_dict.keys()
                    if len(board_dict[box]) > 1]
+
+        # TODO: I might be wrong, but I don't think this does what you're
+        # expecting -- len(board_dict[box]) should always be the same,
+        # maybe you meant len(''.join(board_dict[box]) or something?
```

Comment:
len(board_dict[box]) is the len of the string value at the position key dox in the board dict. So it should only return boxes which have strings longer than 1, and the tests show it does.